addOns/jwt/src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```diff
@@ -0,0 +1,78 @@
+jwt.scanner.client.vulnerability.cookiehttponly.soln=Ensure that the HttpOnly flag is set for all cookies.
+jwt.scanner.client.vulnerability.cookiehttponly.refs=http://www.owasp.org/index.php/HttpOnly
+
+jwt.scanner.client.vulnerability.cookiesecureflag.name=Cookie containing JWT token is without Secure Flag.
+jwt.scanner.client.vulnerability.cookiesecureflag.desc=A cookie has been set without the secure flag, which means that the cookie can be accessed via unencrypted connections.
+jwt.scanner.client.vulnerability.cookiesecureflag.soln=Whenever a cookie contains sensitive information or is a session token, then it should always be passed using an encrypted channel. Ensure that the secure flag is set for cookies containing such sensitive information.
+jwt.scanner.client.vulnerability.cookiesecureflag.refs=http://www.owasp.org/index.php/Testing_for_cookies_attributes_(OWASP-SM-002)
+
+
+jwt.scanner.client.vulnerability.cookiesamesite.name=Cookie containing JWT token is without SameSite Attribute
+jwt.scanner.client.vulnerability.cookiesamesite.desc=A cookie has been set without the SameSite attribute, which means that the cookie can be sent as a result of a 'cross-site' request. \
+The SameSite attribute is an effective counter measure to cross-site request forgery, cross-site script inclusion, and timing attacks.
+jwt.scanner.client.vulnerability.cookiesamesite.soln=Ensure that the SameSite attribute is set to either 'lax' or ideally 'strict' for all cookies.
+jwt.scanner.client.vulnerability.cookiesamesite.refs=https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site
+
+jwt.scanner.client.vulnerability.cookieprefix.name=Cookie containing JWT token is without __Secure- or __Host- prefixes.
+jwt.scanner.client.vulnerability.cookieprefix.desc=A cookie has been set without the  __Secure- or __Host- prefixes, which means that the cookie can be added using Set-Cookie header via unencrypted connections or via malicious subdomains.
+jwt.scanner.client.vulnerability.cookieprefix.soln=Whenever a cookie contains sensitive information or is a session token, ensure hardening the cookie using cookie prefixes.
+jwt.scanner.client.vulnerability.cookieprefix.refs=https://googlechrome.github.io/samples/cookie-prefixes/
+
+jwt.scanner.client.vulnerability.urlparam.name=JWT token is in URL
+jwt.scanner.client.vulnerability.urlparam.desc=The request contains JWT token which is leaked in the URL. This can violate PCI and most organizational compliance policies. You can configure the list of strings for this check to add or remove values specific to your environment.
+jwt.scanner.client.vulnerability.urlparam.soln=Do not pass sensitive information in URIs.
+jwt.scanner.client.vulnerability.urlparam.refs=https://cheatsheetseries.owasp.org/cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.html
+
+jwt.scanner.client.vulnerability.formparam.name=JWT token is in Form.
```

Comment:
```suggestion
jwt.scanner.client.vulnerability.formparam.name=JWT Token Leaked in Form
```

Reply:
i think we should not call it as a leak, this is just a vulnerability type for info so that the developer is sure that he/she has used Local Storage for JWT.

Reply:
Fair enough. But it should be in title caps and not be a sentence, it’s a name.

Maybe 

> Form Contains JWT Token

Or 

> Parameter Value Containing JWT

??

Reply:
Form Contains JWT Token seems more valid. will do the change. thanks @kingthorin